adaptive_app/lib/src/playlist_details.dart

```diff
@@ -0,0 +1,154 @@
+import 'package:flutter/material.dart';
+import 'package:googleapis/youtube/v3.dart';
+import 'package:provider/provider.dart';
+import 'package:url_launcher/link.dart';
+
+import 'adaptive_image.dart';  
+import 'adaptive_text.dart';
+import 'app_state.dart';
+
+class PlaylistDetails extends StatelessWidget {
+  const PlaylistDetails(
+      {required this.playlistId, required this.playlistName, super.key});
+  final String playlistId;
+  final String playlistName;
+
+  @override
+  Widget build(BuildContext context) {
+    return Consumer<FlutterDevPlaylists>(
+      builder: (context, playlists, _) {
+        final playlistItems = playlists.playlistItems(playlistId: playlistId);
+        if (playlistItems.isEmpty) {
+          return const Center(child: CircularProgressIndicator());
+        }
+
+        return _PlaylistDetailsListView(playlistItems: playlistItems);
+      },
+    );
+  }
+}
+
+class _PlaylistDetailsListView extends StatefulWidget {
+  const _PlaylistDetailsListView({required this.playlistItems});
+  final List<PlaylistItem> playlistItems;
+
+  @override
+  State<_PlaylistDetailsListView> createState() =>
```

Comment:
Can you test to change the number of characters per line to try to make this code a single line.
Try to use 100 characters or 120.
We can talk about this offline